WarOfEmpires.Domain.Tests/Alliances/AllianceTests.cs

```diff
@@ -173,5 +173,26 @@ public sealed class AllianceTests {
             alliance.TransferLeadership(member);
             alliance.Leader.Should().Be(member);
         }
+
+        [TestMethod]
+        public void Alliance_Disband_Succeeds() {
+            var leader = new Player(1, "Member");
```

Comment:
It might be more clear if these display names matched the variable names better. It's particularly confusing that the leader's name is Member, when there is also someone who is the member.